src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,22 +2,28 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.core.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger =
+            (Logger) LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called with login: {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
             throw new AuthenticationException("Username or password are incorrect");
         }
+        logger.info("User authenticated successfully with login: {}", login);
         return user;
     }
 
     private User findByLogin(String login) {
         User user = new User(login, "1234");
```

Comment:
The `findByLogin` method is currently hardcoded to return a user with a fixed password. In a real application, this method should query a database or another data source to retrieve the user information.